ports/esp32/Makefile

```diff
@@ -59,29 +65,51 @@ ESPIDF = $(IDF_PATH)
 else
 $(info The ESPIDF variable has not been set, please set it to the root of the esp-idf repository.)
 $(info See README.md for installation instructions.)
-$(info Supported git hash: $(ESPIDF_SUPHASH))
 $(error ESPIDF not set)
+print_supported_git_hash
```

Comment:
This doesn't print the supported hash anymore because this line is after the error line